docs/app/pages/components/components-sections/dashboard/widgets/dashboard-widgets.component.html

```diff
@@ -0,0 +1,224 @@
+        [header]="tableHeader"
+        [data]="tableData">
+    </ux-dashboard-table-widget>
+
+    <ux-dashboard-select-widget
+        class="widget"
+        id="widget-select"
+        name="Select Widget"
+        heading="Select Widget"
+        [fixedMode]="fixedMode"
+        [colSpan]="2"
+        [options]="selectOptions"
+        value="0">
+    </ux-dashboard-select-widget>
+
+    <ux-dashboard-text-widget
+        class="widget"
+        id="widget-text"
+        name="Text Widget"
+        heading="Text Widget"
+        [fixedMode]="fixedMode"
+        [colSpan]="4"
+        [rowSpan]="2"
+        [text]="loremIpsum2"
+        [editable]="true">
+    </ux-dashboard-text-widget>
+
+    <ux-dashboard-text-widget
+        class="widget"
+        id="widget-text-readonly"
+        name="Read-only Text Widget"
+        heading="Read-only Text Widget"
+        [fixedMode]="fixedMode"
+        [colSpan]="4"
+        [rowSpan]="2"
+        [text]="loremIpsum2"
+        [editable]="false">
+    </ux-dashboard-text-widget>
+
+</ux-dashboard>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <ux-accordion>
+            <ux-accordion-panel class="accordion-chevron" heading="Customize Example...">
+                <div class="row">
+                    <div class="col-md-6">
+                        <ux-checkbox [(ngModel)]="fixedMode">fixedMode</ux-checkbox>
+                    </div>
+                    <div class="col-md-2">
+                        <label class="form-label" for="dashboard-padding">Padding</label>
+                    </div>
+                    <div class="col-md-4">
+                        <ux-number-picker-inline id="dashboard-padding" [(value)]="dashboardPadding"
+                                                 min="0" placeholder="Enter padding">
+                        </ux-number-picker-inline>
+                    </div>
+                </div>
+            </ux-accordion-panel>
+        </ux-accordion>
+    </div>
+</div>
+
+<hr>
+
+<p>
+    The pre-built widgets which include the <code>ux-dashboard-actions-widget</code>,
+    <code>ux-dashboard-select-widget</code>,
+    <code>ux-dashboard-table-widget</code> and <code>ux-dashboard-text-widget</code> are ready-to-use widgets for the
+    <code>ux-dashboard</code>
+    component.
+</p>
+
+<p>The following attributes are required by all pre-built widgets:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="id" type="string" [required]="true">The id given to the widget.</tr>
+    <tr uxd-api-property name="name" type="string" [required]="true">The name used by aria labels and announcements.
+    </tr>
+    <tr uxd-api-property name="heading" type="string">The heading displayed for the widget.</tr>
+    <tr uxd-api-property name="fixedMode" type="boolean">Whether the widget can be dragged and resized.</tr>
+    <tr uxd-api-property name="colSpan" type="number">The number of columns the widget spans.</tr>
```

Comment:
We should keep these inline with the dashboard descriptions for example. "Defines the number of columns this widget should occupy."